e2e-tests/insurance/cypress/e2e/journeys/policy/broker/save-and-back.spec.js

```diff
@@ -103,6 +104,8 @@ context('Insurance - Policy - Broker page - Save and back', () => {
       submitButton().click();
       // submit single contract policy form
       submitButton().click();
+      // submit total contract value form
+      submitButton().click();
```

Comment:
I've had an idea.  What if we make a cypress function with a for loop and we loop the submitButton click so we don't need to paste it many times

Reply:
Great idea, I was thinking of creating something like `partiallySubmitXSection({ ... })` but a submission counter loop sounds loads better.

Can we do this as a piece of tech debt, we have lots of instances of submit buttons would need to go through a lot of files.